treeherder/webapp/api/refdata.py

```diff
@@ -56,11 +56,3 @@ class UserViewSet(viewsets.ReadOnlyModelViewSet):
 
     def get_queryset(self):
         return User.objects.filter(id=self.request.user.id)
-
-
-class MatcherViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = models.Matcher.objects.all()
```

Comment:
It looks like the `Matcher` model is unused now - other than in this migration (which could be made a no-op?):
https://github.com/mozilla/treeherder/blob/ffaa2e4b2af200d3ae91e3e6ecdc2d4e4f093e84/treeherder/model/migrations/0004_populate_matcher_name_fields.py